clusterloader2/pkg/prometheus/manifests/kubemark/kubemark-master-endpoints.yaml

```diff
@@ -18,3 +18,5 @@ subsets:
         port: 10251
       - name: kube-controller-manager
         port: 10252
+      - name: etcd
+        port: 2379
```

Comment:
Unfortunately, it won't work just like that.
It looks like the etcd listens on port 2379 only on localhost - https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/etcd.manifest#L26
```
--advertise-client-urls http://127.0.0.1:{{ port }}
```

Reply:
As discussed offline, it works in kubemark as kubemark uses differet etcd config with --listen-client-urls=http://0.0.0.0:2379

Reply:
After discussing this personally, looks like it will work for kubemark. When setting up etcd for kubemark master we use
```
-listen-client-urls=http://0.0.0.0:2379 
```